tests/integration/cattletest/core/test_container_logs.py

```diff
@@ -0,0 +1,40 @@
+from common_fixtures import *  # NOQA
+from test_docker import docker_context, TEST_IMAGE_UUID, if_docker
+
+# work around flake8 issue
+docker_context
+
+
+def _get_container_logs_ip(host):
+    found_ip = None
+    for ip in host.ipAddresses():
+        if found_ip is None:
+            found_ip = ip
+        elif found_ip.role == 'primary':
+            found_ip = ip
+            break
+        elif ip.createdTS < found_ip.createdTS:
+            found_ip = ip
+
+    assert found_ip is not None
+    assert found_ip.address is not None
+
+    return found_ip
+
+
+@if_docker
+def test_logs_container(admin_client):
```

Comment:
If using docker always depend on the `docker_context` fixture.  This ensures that docker is properly setup.